cassandane/tiny-tests/JMAPCalendars/calendar_set_defaultalerts_shared

```diff
@@ -134,10 +137,10 @@ sub test_calendar_set_defaultalerts_shared
         }, 'R1'],
     ]);
 
-    $self->assert_shared_defaultalerts($t);
+    $self->assert_shared_defaultalerts($t, 'no alarm!', 'no alarm!');
 }
 
-sub _can_match {
+sub _can_match_b {
```

Comment:
Are these two functions identical?  In that case I would've just moved it up to the parent.  The fact that you gave them distinct names instead makes me think maybe they were distinct functions sharing a name, but the bodies don't show in the diff and I didn't look any deeper to find out

Reply:
They are identical and also appear in cassandane/Cassandane/Cyrus/CaldavAlarm.pm